run-integration-tests-local.sh

```diff
@@ -9,11 +9,14 @@
 #
 # As the script will add a generated CA certificate to the local OS it needs root or a password to run.
 
-export DOCKER_HOSTNAME=kind-registry:5000
-export DOCKER_NAMESPACE=m4d-system
+set -x
+set -e
+
+export DOCKER_HOSTNAME=${DOCKER_HOSTNAME:=kind-registry:5000}
+export DOCKER_NAMESPACE=${DOCKER_NAMESPACE:=m4d-system}
 export HELM_EXPERIMENTAL_OCI=1
 
-make kind-setup
+make kind
```

Comment:
Why do you upgrade this to kind? Do we need istio somewhere? This will increase the runtime quite a bit..

Reply:
opened a related issue https://github.com/IBM/the-mesh-for-data/issues/132

Reply:
ok, will revert change, and I have assigned myself to the issue